boards/adafruit/feather_esp32s2/adafruit_feather_tft_esp32s2.dts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2025 Philipp Steiner <philipp.steiner@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include "adafruit_feather_esp32s2_common.dtsi"
+#include "feather_connector_tft.dtsi"
+
+/ {
+	model = "Adafruit Feather ESP32-S2 TFT";
+	compatible = "espressif,esp32s2";
+
+	chosen {
+		zephyr,display = &st7789v_tft;
```

Comment:
Does it require setting console, flash and so? I haven't fully checked the board architecture.

```c
zephyr,console = &uart1;
zephyr,shell-uart = &uart1;
zephyr,flash = &flash0;
```

Reply:
I see now it is a feather board..

Reply:
On other boards like the ESP32-S2-DevKitC, ESP32-S2-Saola and other EPS32-S2 Boards does it, so I did it too.